src/radius/radius.h

```diff
@@ -124,7 +133,11 @@ enum { RADIUS_ATTR_USER_NAME = 1,
 #define RADIUS_TERMINATION_ACTION_RADIUS_REQUEST 1
 
 /* NAS-Port-Type */
+#ifdef CONFIG_SONIC_RADIUS
+#define RADIUS_NAS_PORT_TYPE_IEEE_802_11 15
```

Comment:
IEEE 802.11 standard wireless interface has NAS port type as 19, and for "Ethernet" interface its 15, can we have change remove the below instead "ifdef -else" block"
 
 #ifdef CONFIG_SONIC_RADIUS
#define RADIUS_NAS_PORT_TYPE_ETHERNET 15
#endif